cordformation/src/test/resources/gradle.properties

```diff
@@ -3,4 +3,5 @@ org.gradle.jvmargs=-XX:+UseG1GC -Xmx512m
 org.gradle.caching=false
 corda_group=net.corda
 corda_release_version=4.0
-jolokia_version=1.6.0
\ No newline at end of file
+jolokia_version=1.6.0
+docker_image_name=corda/corda-zulu-java1.8-4.3
```

Comment:
What's worrying me here is that `docker_image_name` is _really_
```
corda/corda-zulu-java1.8-$corda_release_version
```
but `corda_release_version` is still `4.0`. So these properties are not consistent. Even worse, this test is probably deploying CorDapps from 4.0 into a Corda 4.3 Docker image. Think about how you could make these tests consistent.

Reply:
I'm changing this to `corda/corda-zulu-4.0` to highlight the consistency between the Corda Docker image and Corda version values. While this Docker image name points to a valid Docker image on DockerHub, the tests are to ensure that the dockerImage tag must provided as part of the Dockerform task.